src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -138,10 +159,52 @@ protected void SendKeys(params object[] keys)
             => Editor.SendKeys(keys);
 
         protected void DisableSuggestionMode()
-            => VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(false);
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(false);
+            WaitForAsyncOperations(FeatureAttribute.Workspace);
+        }
 
         protected void EnableSuggestionMode()
-            => VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(true);
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(true);
+            WaitForAsyncOperations(FeatureAttribute.Workspace);
+        }
+
+        protected void EnableQuickInfo()
```

Comment:
No. Do not move all helper methods into `AbstractEditorTest`. I'm not a fan of the "dumping ground" design pattern. :-(

We shouldn't be using inheritance as a means of supplying _access_ to useful functionality. Code that is broadly useful should be grouped into classes of related functionality, like we've done with `SolutionExplorer_In\OutOfProc` for manipulating solutions and projects, the dialog types for manipulating specific dialogs, etc. Inheritance should be used to _customize_ behavior, like when we need each test class to supply the appropriate language name.

This strongly suggests that we shouldn't have an `AbstractEditorTest` class at all: most functionality should be moved out to helper types, or perhaps down to `AbstractIntegrationTest` itself.